blog/source/_posts/Publishing-your-first-boardgame-io-game-in-FBG.md

```diff
@@ -153,13 +152,13 @@ export const CheckersGame = Game({
 Now you have a skeleton of a game in your new folder, but you still need to add the new game to the home page and to the server. This is easily done with three lines of code. Add the following to the first lines of `src/games/index.ts`:
 
 ```typescript
-import { checkersGameDef } from './checkers';
+import { fooBarGameDef } from './foobar';
 ```
 
-Then, in the same file, under `GAMES_MAP`, add `checkers: checkersGameDef,` and under `GAMES_LIST`, add `GAMES_MAP.checkers,`.
+Then, in the same file, under `GAMES_MAP`, add `foobar: fooBarGameDef,` and under `GAMES_LIST`, add `GAMES_MAP.fooBar,`.
 
 Done! You can run now `yarn run dev` and you should be able to see your new game on the home page! It will have two game modes, one for playing with friends locally and another for playing with friends over the internet. Because we have `debug: true` in `config.ts`, you will be able to see the boardgame.io debug menu.
 
-There are great boardgame.io [tutorials and documentation](https://boardgame.io/#/tutorial); you can follow any of them to get started on how to create your first game. Also, feel free to look into other FBG games and see how they are implemented. Finally, if you get stuck with some issue, send a message to our comunity!  We will do our best to help you out :).
+There are great boardgame.io [tutorials and documentation](https://boardgame.io/#/tutorial); you can follow any of them to get started on how to create your first game. Also, feel free to look into other FBG games and see how they are implemented. Finally, if you get stuck with some issue, send a message to our comunity! We will do our best to help you out :).
```

Comment:
Even though a period at the end is grammatically correct, I think it looks weird with the smiley. What do you think?

Reply:
Whatever, I didnt touch this in the PR